### PR TITLE
SED-2890-fix issue

### PR DESCRIPTION
**Pull request by lucianbaciu**
fixed in both 24 and 25 branches

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts [hunk 1]
```diff
@@ -71,27 +71,39 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
...
-      // this.bucketsByKeywords[selection.id].attributes!['isSelected'] = selection.isSelected;
-      const entries = this.tableData$.getValue();
-      entries.forEach((entry) => {
-        if (entry.name === selection.id) {
-          entry.isSelected = selection.isSelected;
+    keywordsContext
+      .onKeywordToggled()
+      .pipe(takeUntil(this.terminator$))
+      .subscribe((selection) => {
+        // this.bucketsByKeywords[selection.id].attributes!['isSelected'] = selection.isSelected;
```
Comment (neogucky) [inline projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts:78]:
```suggestion
```

File: projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts [hunk 2]
```diff
@@ -71,27 +71,39 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
     this.tableDataSource = new TableLocalDataSource(this.tableData$, this.getDatasourceConfig());
     this.keywordsService = this.executionContext.keywordsContext;
     const keywordsContext = this.executionContext.keywordsContext;
```
Comment (neogucky) [inline projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts:73]: Why are we saving the context in a class varaible as well as in a local const?

Comment (lucianbaciu) [reply projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts:73]: for no reason

File: projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts (hunk 1 above)
Comment (neogucky) [reply projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts:77]: @lucianbaciu I saw you updated the PR but left the comment

Comment (neogucky) [reply projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts:77]: I just accepted this minor change myself to approve the PR

**Review by neogucky — APPROVED** [review]

**Pull request merged**